images/tidb-operator/Dockerfile

```diff
@@ -1,8 +1,21 @@
 FROM alpine:3.5
 
+ARG KUBE_VERSION=1.12.10
```

Comment:
Why package `kubectl` in tidb-operator Docker image? This increases the image size and binds specific `kubectl` to tidb-operator, besides it adds extra attack surface for tidb-operator. What if there is no code change but kubectl has a security bugfix, shall we update the docker image and tell user to upgrade tidb-operator image?

Reply:
> Why package `kubectl` in tidb-operator Docker image? This increases the image size and binds specific `kubectl` to tidb-operator, besides it adds extra attack surface for tidb-operator. What if there is no code change but kubectl has a security bugfix, shall we update the docker image and tell user to upgrade tidb-operator image?

Right, I think installing openssl would be necessary here. I will remove `kubectl` and edit the scripts.

Reply:
fix